test_rclcpp/CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ if(BUILD_TESTING)
       TIMEOUT 15)
     custom_gtest(gtest_executor
       "test/test_executor.cpp"
-      TIMEOUT 30)
+      TIMEOUT 60)
```

Comment:
it was mentioned in the meta-ticket where the CI jobs were referenced: https://github.com/ros2/rcl/pull/137#issuecomment-305034192